src/models/Room.models.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../db.js';
+
+export const Room = sequelize.define(
+  'room',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    roomName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: 'room',
+    updatedAt: false,
+    createdAt: false,
+  },
+);
+
+Room.sync({force: true});
```

Comment:
This line of code forces the table to be dropped and then created again every time the application starts. This is not advisable for a production environment as it would lead to data loss each time the application is restarted. It's better to control your database schema changes using migrations. If you are using Sequelize, you can use its CLI to generate and run migrations. If you are in a development environment and you are sure about what you are doing, then it's okay to use it, but remember to remove or comment it out before moving to production.